src/components/DescriptionList/_description-list.scss

```diff
@@ -0,0 +1,34 @@
+.cc-description-list__title {
+  display: block;
+  float: left;
+  font-size: var(--body-lg);
+  font-weight: bold;
+
+  margin: 0 var(--space-sm)  0 0;
+}
+
+.cc-description-list__definition {
+  display: block;
+  margin: 0 0 var(--space-sm) 0;
+
+  a {
+    @extend %anchor-base;
+  }
+
+  p {
+    margin: 0;
+  }
+
+  span {
+    display: inline;
+  }
+
+  span:after {
+    content: ', ';
+    display: inline;
+  }
+
+  span:last-child:after {
+    content: '';
+  }
```

Comment:
Is this due to "legacy" markup being migrated from D7?

Reply:
I think I'll be able to remove this styling if I used the `RichText` component.

Reply:
😎